cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java

```diff
@@ -586,9 +627,16 @@ public int getReplicas() {
     }
 
     /**
-     * @return  Logging Model instance for configuring logging
+     * @return Logging Model instance for configuring logging
```

Comment:
Having less various unnecessary whitespace changes would make the review much easier. It is especially wierd if you remove tabs from the Javadocs in one place and use them in new Javadocs in other place.

Reply:
Agreed. Reverted.